ros/src/twist_controller/dbw_node.py

```diff
@@ -53,25 +56,35 @@ def __init__(self):
         self.brake_pub = rospy.Publisher('/vehicle/brake_cmd',
                                          BrakeCmd, queue_size=1)
 
-        # TODO: Create `Controller` object
-        # self.controller = Controller(<Arguments you wish to provide>)
-
-        # TODO: Subscribe to all the topics you need to
+        self.dbw_enabled = False
+        self.current_velocity = None
+        self.twist_cmd = None
+
+        # Create `TwistController` object
+        self.controller = Controller(
+                wheel_base = wheel_base,
+                steer_ratio = steer_ratio,
+                min_speed = 0.5,
```

Comment:
Should we also add min_speed to parameter server?

Reply:
Makes sense